core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -2363,4 +2363,12 @@ package object config {
       .version("3.3.0")
       .intConf
       .createWithDefault(5)
+
+  val SPARK_PROPERTIES_KEY_QUEUE = ConfigBuilder("spark.properties.key.queue")
+    .doc("Specifies the spark property key name for queue information." +
+      " When it is specified, the eventLog parser could use it as the key" +
+      " to extract the queue value from the Spark Properties in the eventLog files.")
+    .version("3.3.0")
+    .stringConf
+    .createWithDefault("spark.yarn.queue")
```

Comment:
What other value is possible here?

Reply:
Same as above, the configuration is only default to `spark.yarn.queue` which is the spark properties key name for queue in Yarn, but can be overridden as a spark conf if spark is running on other platforms. For example, when spark runs on K8s, the queue info may be specified as a pod annotation which is used by a scheduler to schedule jobs to the corresponding queues in the cluster. A example of such queue property key name in k8s could be spark.kubernetes.driver.label.queue